gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -231,8 +293,53 @@ static void
 gst_tensor_if_set_property (GObject * object, guint prop_id,
     const GValue * value, GParamSpec * pspec)
 {
-  /* GstTensorIf *self = GST_TENSOR_IF (object); */
-  /** @todo NYI! */
+  GstTensorIf *self = GST_TENSOR_IF (object);
+
+  g_critical ("Setting property for prop %d.\n", prop_id);
+
+  switch (prop_id) {
+    case PROP_CV:
+      self->cv = g_value_get_enum (value);
+      g_critical ("Compared value : %d", self->cv);
+      break;
+    case PROP_CV_OPTION:
+    {
+      self->cv_option = gst_tensor_if_set_property_glist (value, self->cv_option, ":,");
+      break;
+    }
+    case PROP_OP:
+      self->op = g_value_get_enum (value);
+      g_critical ("operator : %d", self->op);
+      break;
+    case PROP_SV:
+    {
+      self->sv = gst_tensor_if_set_property_glist (value, self->sv, ",");
+      break;
+    }
+    case PROP_SV_OPTION:
+      break;
+    case PROP_THEN:
+      self->act_then = g_value_get_enum (value);
+      g_critical ("Set act_then = %d", self->act_then);
+      break;
+    case PROP_THEN_OPTION:
+      self->then_option = gst_tensor_if_set_property_glist (value, self->then_option, ",");
+      break;
+    case PROP_ELSE:
+      self->act_else = g_value_get_enum (value);
+      g_critical ("Set act_else = %d", self->act_else);
+      break;
+    case PROP_ELSE_OPTION:
+      self->else_option = gst_tensor_if_set_property_glist (value, self->else_option, ",");
+      break;
+    case PROP_SILENT:
+      self->silent = g_value_get_boolean (value);
+      g_critical ("Set silent = %d", self->silent);
+      break;
+    default:
+      G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, pspec);
+      break;
+  }
```

Comment:
How are you going to react if set_prop was called while the pipeline is running? or paused?

Reply:
I think it would be right to allow changing property during running or paused only for the `supplied-value`.
I will change it later.